pibooth/view/window.py

```diff
@@ -248,7 +248,7 @@ def show_intro(self, pil_image=None, with_print=True):
         else:
             self._update_background(background.IntroBackground(self.arrow_location, self.arrow_offset))
 
-        if pil_image:
+        if with_print and pil_image:
```

Comment:
Why adding this criteria?

Reply:
I need this test for the fullscreen intro screen, without it, the text and the previous image will overlap together